packages/fuselage/src/components/Callout/Callout.stories.tsx

```diff
@@ -36,36 +37,53 @@ export default {
 } as ComponentMeta<typeof Callout>;
 
 const Template: ComponentStory<typeof Callout> = (args) => (
-  <>
-    <Callout title='This is a generic message'>{args.children}</Callout>
-    <Callout title='This is a info message' type='info'>
-      {args.children}
-    </Callout>
-    <Callout title='This is a successful message' type='success'>
-      {args.children}
-    </Callout>
-    <Callout title='This is a warning message' type='warning'>
-      {args.children}
-    </Callout>
-    <Callout title='This is a danger message' type='danger'>
-      {args.children}
-    </Callout>
-  </>
+  <Callout {...args} />
```

Comment:
Since all stories are going to have the same `children`, I would suggest adding it to the template.

Reply:
You are right, I agree.